src/utils/use-interval.ts

```diff
@@ -1,13 +1,14 @@
 import {useRef, useEffect} from 'react';
 
```

Comment:
Should e.g. https://developer.mozilla.org/en-US/docs/Web/API/setInterval#ensure_that_execution_duration_is_shorter_than_interval_frequency be added in a comment for future reference to explain why setInterval isn't used instead?